cook-book/getting-started/README.md

```diff
@@ -0,0 +1,287 @@
+at once.
+We will start more instances, add them to topology and edit cluster
+configuration step by step.
+
+Since version [1.8.0](https://github.com/tarantool/ansible-cartridge/releases/tag/1.8.0)
+`tarantool.cartridge` role allows to specify [scenario](/doc/scenario.md)
+that consists of steps.
+The role has pre-defined [steps](/doc/scenario.md#steps), but you can also
+[write your own step](/doc/scenario.md#adding-custom-step-to-scenario) and
+use it in scenario.
+
+Using scenarios allows to perform different actions with cluster separately.
+
+The role has several common [scenarios](/doc/scenario.md#scenarios) rhat
+can be re-used.
+It's also possible to write custom scenario and then use it in different plays.
+
+### Change configuration
+
+Now we have such topology:
+
+* `storage-1` replicaset (`vshard-storage`):
+  * `storage-1-leader`
+  * `storage-1-replica`
+
+Let's add `core-1` replicaset and add one more replica to `storage-1` replicaset:
+
+* `core-1` replicaset (`vshard-router`, `failover-coordinator`):
+  * `core-1`
+
+* `storage-1` replicaset (`vshard-storage`):
+  * `storage-1-leader`
+  * `storage-1-replica`
+  * `storage-1-replica-2`
+
+Be sure that you've read
+[configuration basics guide](/doc/configuration-basics.md)
+to understand why do we change inventory this way.
+
+Change [inventory](./hosts.yml):
+
+```diff
+@@ -23,6 +23,16 @@ all:
+         advertise_uri: '172.19.0.2:3302'
+         http_port: 8182
+
++    storage-1-replica-2:
++      config:
++        advertise_uri: '172.19.0.3:3304'
++        http_port: 8184
++
++    core-1:
++      config:
++        advertise_uri: '172.19.0.2:3301'
++        http_port: 8181
++
+   children:
+     # group instances by machines
+     machine_1:
+@@ -32,6 +42,7 @@ all:
+
+       hosts:
+         storage-1-replica:
++        core-1:
+
+     machine_2:
+       vars:
+@@ -40,12 +51,14 @@ all:
+
+       hosts:
+         storage-1:
++        storage-1-replica-2:
+
+     # group instances by replicasets
+     replicaset_storage_1:
+       hosts:
+         storage-1:
+         storage-1-replica:
++        storage-1-replica-2:
+       vars:
+         replicaset_alias: storage-1
+         failover_priority:
+@@ -53,3 +66,12 @@ all:
+           - storage-1-replica
+         roles:
+           - vshard-storage
++
++    replicaset_core_1:
++      hosts:
++        core-1:
++      vars:
++        replicaset_alias: core-1
++        roles:
++          - vshard-router
++          - failover-coordinator
+```
+
+### Configure instances
+
+To start and configure instance we can use
+[`configure_instances`](/doc/scenario.md#scenarios) pre-defined scenario.
+But it also contain steps we don't need (like `deliver_package` and `update_package`).
+`update_instance` step is used only with [multiversion approach](/doc/multiversion.md)
+, so it can be removed too.
+
+Use this `cartridge_scenario` in [playbook.configure-instances.yml](./playbook.configure-instances.yml):
```

Comment:
But this scenario already used in this file... 

Maybe "we use this..." or "Use ... You should get file as ..."